src/System.Management.Automation/FormatAndOutput/DefaultFormatters/FileSystem_format_ps1xml.cs

```diff
@@ -11,11 +11,10 @@ internal static IEnumerable<ExtendedTypeDefinition> GetFormatData()
         {
```

Comment:
@powercode, your last commit had 3 failures in `PowerShell-CI-macos`
Basic FileSystem Provider Tests.Validate basic FileSystem Cmdlets.Get-Content on Unix succeeds with folder and file with colon: \foo:bar.txt
```powershell
Expected exactly 'Hello', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 246
246:                 $files[0] | Get-Content | Should -BeExactly "Hello"
```
Basic FileSystem Provider Tests.Validate basic FileSystem Cmdlets.Get-Content on Unix succeeds with folder and file with colon: /foo:
```powershell
Expected exactly 'Hello', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 246
246:                 $files[0] | Get-Content | Should -BeExactly "Hello"
```
Basic FileSystem Provider Tests.Validate basic FileSystem Cmdlets.Get-Content on Unix succeeds with folder and file with colon: \:bar
```powershell
Expected exactly 'Hello', but got $null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1: line 246
246:                 $files[0] | Get-Content | Should -BeExactly "Hello"
```